mapper/orm/pom.xml

```diff
@@ -60,6 +60,26 @@
             <!-- DO NOT REMOVE and DO NOT MANAGE the version of this dependency. See the version property declaration. -->
             <version>${version.org.hibernate.commons.annotations}</version>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye</groupId>
+            <artifactId>jandex</artifactId>
+            <!-- DO NOT REMOVE and DO NOT MANAGE the version of this dependency. See the version property declaration. -->
+            <version>${version.io.smallrye.jandex}</version>
+        </dependency>
+        <!-- Solve dependency convergence issues through exclusions for transitive-only dependencies -->
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <!-- DO NOT REMOVE and DO NOT MANAGE the version of this dependency. See the version property declaration. -->
+            <version>${version.jakarta.xml.bind}</version>
+            <exclusions>
+                <!-- Also imported by org.glassfish.jaxb:jaxb-core and its transitive dependency org.eclipse.angus:angus-activation -->
+                <exclusion>
+                    <groupId>jakarta.activation</groupId>
+                    <artifactId>jakarta.activation-api</artifactId>
+                </exclusion>
```

Comment:
Shouldn't we rather exclude the dependency from jaxb-core?
That's what we did for Jakarta artifacts in Search 6: https://github.com/hibernate/hibernate-search/blob/8a75143253ace18e18e55ac31cddd1c428446983/jakarta/build/parents/public/pom.xml#L49-L79

BTW we can use dependency management in the root POM for this. We just can't use a version in the dependency management entry.

Reply:
so we are ending up with this:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.3.0:enforce (enforce-dependency-convergence) on project hibernate-search-mapper-orm: 
[ERROR] Rule 0: org.apache.maven.enforcer.rules.dependency.DependencyConvergence failed with message:
[ERROR] Failed while enforcing releasability.
[ERROR] 
[ERROR] Dependency convergence error for jakarta.activation:jakarta.activation-api:jar:2.1.0 paths to dependency are:
[ERROR] +-org.hibernate.search:hibernate-search-mapper-orm:jar:7.0.0-SNAPSHOT
[ERROR]   +-org.hibernate.orm:hibernate-core:jar:6.2.5.Final:compile
[ERROR]     +-jakarta.xml.bind:jakarta.xml.bind-api:jar:4.0.0:runtime
[ERROR]       +-jakarta.activation:jakarta.activation-api:jar:2.1.0:runtime
[ERROR] and
[ERROR] +-org.hibernate.search:hibernate-search-mapper-orm:jar:7.0.0-SNAPSHOT
[ERROR]   +-org.hibernate.orm:hibernate-core:jar:6.2.5.Final:compile
[ERROR]     +-org.glassfish.jaxb:jaxb-runtime:jar:4.0.2:runtime
[ERROR]       +-org.glassfish.jaxb:jaxb-core:jar:4.0.2:runtime
[ERROR]         +-jakarta.activation:jakarta.activation-api:jar:2.1.1:runtime
[ERROR] and
[ERROR] +-org.hibernate.search:hibernate-search-mapper-orm:jar:7.0.0-SNAPSHOT
[ERROR]   +-org.hibernate.orm:hibernate-core:jar:6.2.5.Final:compile
[ERROR]     +-org.glassfish.jaxb:jaxb-runtime:jar:4.0.2:runtime
[ERROR]       +-org.glassfish.jaxb:jaxb-core:jar:4.0.2:runtime
[ERROR]         +-org.eclipse.angus:angus-activation:jar:2.0.0:runtime
[ERROR]           +-jakarta.activation:jakarta.activation-api:jar:2.1.1:runtime
[ERROR] 
```
I thought I'd exclude it from API one since there's a 2.1.0, while the core one is using 2.1.1

Reply:
I don't know... it seems wrong to me that jaxb-core would override the version of Jakarta dependencies... I'll let you decide, but please do that in the root POM if possible.

BTW, when we will just import BOMs, this whole mess wouldn't be ours to solve. Hopefully we'll be able to do that soon (in another PR, though). There's https://hibernate.atlassian.net/browse/HSEARCH-4699 to use Hibernate ORM's [`hibernate-platform`](https://central.sonatype.com/artifact/org.hibernate.orm/hibernate-platform/6.2.5.Final/overview). For Jakarta there's [this one](https://mvnrepository.com/artifact/jakarta.platform/jakarta.jakartaee-bom) and maybe others, but I'd rather not use it before Hibernate ORM does.